builds/e2e/templates/connectivity-deploy.yaml

```diff
@@ -52,6 +54,16 @@ steps:
         . $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/artifactInfo.txt
         chmod +x $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/scripts/testHelper.sh
         chmod +x $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/scripts/connectivityTest.sh
+
+        testInfo="TestBuildNumber=${{ parameters['test.buildNumber'] }}"
+        testInfo="$testInfo,TestBuildId=${{ parameters['test.buildId'] }}"
+        testInfo="$testInfo,BuildSourceBranch=${{ parameters['build.source.branch'] }}"
+        testInfo="$testInfo,EdgeletSourceBranch=${{ parameters['edgelet.source.branch'] }}"
+        testInfo="$testInfo,ImagesSourceBranch=${{ parameters['images.source.branch'] }}"
+        testInfo="$testInfo,HostPlatform=${{ parameters['metricsCollector.hostPlatform'] }}"
+        testInfo="$testInfo,RunProfile=${{ parameters['networkController.runProfile'] }}"
+        testInfo="$testInfo,CustomEdgeAgentImage=${{ parameters['customEdgeAgent.image'] }:-None}"
```

Comment:
how about edgeRuntime.buildNumber pipeline variable?

Reply:
I don't think I needed so I didn't add.

Reply:
Then why do you need CustomEdgeAgentImage and CustomEdgeHubImage, but not edgeRuntime.buildNumber? edgeRuntime.buildNumber is used to control connectivity test to run with a specific docker image from other build instead of latest build from given branches.

Reply:
Oh I misunderstood. I'll add.

Reply:
I misunderstood what the custom image was, but the reason I added though was to be able to filter when someone runs connectivity tests for 109. With just the build number, we don't have any information to know that connectivity tests came from the 109 branch. We will need some identifying information about what branch, however if we eventually plan to backport the connectivity tests then we don't have to fix. What do you think?

Reply:
I think we should leave it as is and backport the tests.

Reply:
1. I would just to get this PR committed first; and have another PR to address this problem.
2. To address this problem, are you going to add 1 more field and add people to update that new field when edgeRuntime.buildNumber is set?
3. Or we should just add to get edgeRuntime.SourceBranch instead of edgeRuntime.buildNumber; and use source branch to find the corresponding build number?

Reply:
I think we should do 3 only if we think that we will never backport the tests. If we plan on backporting the tests, then we won't need to support this hack anymore.